src/markdown-pages/contribute-to-quickstarts/build-a-quickstart/add-data-source.mdx

```diff
@@ -0,0 +1,108 @@
+---
+title: 'Add a data source to the quickstart'
```

Comment:
@mehreentahir16 , This might need further conversation. Based on my last discussion with the I/O team, the datasource can't be added until after the qs is live, so that it can link to the quickstart via the id.
cc @aswanson-nr

Reply:
Yeah, third party data-sources need a reference back to the quickstart to know what to install. So unfortunately, you have to submit the quickstart first, get an ID generated, then create the data-source. We know this is a pain point and it will eventually change, but I don't have a timeline on that yet.